setup.py

```diff
@@ -57,6 +57,7 @@
     "exporters": ["onnx<1.14.0", "onnxruntime", "timm"],
     "exporters-gpu": ["onnx<1.14.0", "onnxruntime-gpu", "timm"],
     "exporters-tf": ["tensorflow>=2.4,<2.11", "tf2onnx", "onnx", "onnxruntime", "timm", "h5py", "numpy<1.24.0"],
+    "diffusers": ["diffusers"],
```

Comment:
Do we test anywhere in the stable diffusion ONNX pipelines that diffusers is available (if needed)?

Reply:
Yes we do : https://github.com/huggingface/optimum/blob/add-img2img/optimum/onnxruntime/__init__.py#L76 if `diffusers` is not installed when instantiating an `ORTStableDiffusionPipeline` an `ImportError` will be raised